src/App.css

```diff
@@ -11,11 +11,14 @@
     --wrong-network-border-hover: #C2185B;
     --wrong-network-hover: #E91E63;
     --wrong-network: #EC407A;
+    --red-color: #ff0000;
     --light-text-gray: #bdbdbd;
     --text-gray-onHover: #727D82;
     --dark-text-gray: #616161;
     --nav-text-light: rgba(55, 71, 79, 0.4);
     --nav-text-dark: rgba(55, 71, 79, 1);
+    --light-black:#2b2b2b;
```

Comment:
There should be a space after the colon, same for all the other variables that are like this